src/app/timer/timer.component.ts

```diff
@@ -1,8 +1,32 @@
-import { Component } from '@angular/core';
+import { GameManagerService } from './../game-manager/game-manager.service';
+import { TimerService } from './../shared/timer/timer.service';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Subscription } from 'rxjs';
 
 @Component({
   selector: 'app-timer',
   templateUrl: './timer.component.html',
   styleUrls: ['./timer.component.css'],
 })
-export class TimerComponent {}
+export class TimerComponent implements OnInit, OnDestroy {
+  timer = 0;
+  timerSub!: Subscription;
+  gameWillStartSoon = false;
+
+  constructor(
+    private timerService: TimerService,
+    private gameManager: GameManagerService,
+  ) {}
+
+  ngOnInit(): void {
+    this.gameWillStartSoon = this.gameManager.gameWillStartSoon;
+    this.gameManager.refreshCurrentGameInfo();
+    this.timerSub = this.timerService.runningTimer.subscribe((time) => {
```

Comment:
use async pipe instead of subscription, it is better in that use-case

Reply:
done